### PR TITLE
fix: fixed focus handling, focus now jumps from header to panel when …

### PR DESCRIPTION
**Pull request by DannyMoerkerke**
…tabbing

## What I did

1. after the slotted invokers and content sections are set up they are reordered into `<div class="accordion">` to fix the focus handling. Focus now jumps from header to panel.

todo: fix tests

### CODE REVIEW

File: packages/accordion/src/LionAccordion.js [hunk 1]
```diff
@@ -180,12 +202,33 @@ export class LionAccordion extends LitElement {
...
+   *  @private
+   */
+  __rearrangeInvokersAndContent() {
+    const invokers = /** @type {HTMLElement[]} */ (
+      Array.from(this.querySelectorAll('[slot="invoker"]'))
+    );
+    const contents = /** @type {HTMLElement[]} */ (
+      Array.from(this.querySelectorAll('[slot="content"]'))
+    );
+    const accordion = this.shadowRoot?.querySelector('slot[name=accordion]');
```
Comment (tlouisse) [inline packages/accordion/src/LionAccordion.js:222]: why would/could this be null? Maybe you can do `/** @type {ShadowRoot} */ (this.shadowRoot)`

Comment (DannyMoerkerke) [reply packages/accordion/src/LionAccordion.js:231]: I can but I followed the convention in this file which is currently `this.shadowRoot?`.

File: packages/accordion/src/LionAccordion.js [hunk 2]
```diff
@@ -43,21 +43,40 @@ export class LionAccordion extends LitElement {
...
           margin: 0;
           visibility: hidden;
           display: none;
         }
 
-        .accordion ::slotted([slot='content'][expanded]) {
+        .accordion ::slotted([slot='accordion'][expanded]) {
+          visibility: visible;
+          display: block;
+        }
```
Comment (gerjanvangeest) [inline packages/accordion/src/LionAccordion.js:63]: All of this styling  can be gone if i'm correct.

File: packages/accordion/src/LionAccordion.js [hunk 3]
```diff
@@ -129,6 +148,7 @@ export class LionAccordion extends LitElement {
       <div class="accordion">
         <slot name="invoker"></slot>
         <slot name="content"></slot>
+        <slot name="accordion"></slot>
```
Comment (gerjanvangeest) [inline packages/accordion/src/LionAccordion.js:151]: Am i correct that this slot should not be used by our users, so its basically a protected thing. Maybe call it `_accordion`?

File: packages/accordion/src/LionAccordion.js (hunk 2 above)
Comment (DannyMoerkerke) [reply packages/accordion/src/LionAccordion.js]: Correct, I removed it.

File: packages/accordion/src/LionAccordion.js (hunk 3 above)
Comment (DannyMoerkerke) [reply packages/accordion/src/LionAccordion.js]: Updated.

**Review by gerjanvangeest — APPROVED** [review]

**Pull request merged**